cli/requirements.txt

```diff
@@ -3,5 +3,6 @@ tox>=2.2, <3.0
 wheel>=0.24.0, <1.0
 pytest>=2.9.1
 teamcity-messages>=1.20
+retrying
```

Comment:
Could you either specify the exact version or a range so that it sticks to the current major version? This would make sure that in the future we won't jump unexpectedly to the next major version and get breakages.